lookout/style/typos/analyzer.py

```diff
@@ -308,18 +377,25 @@ def check_identifiers(self, identifiers: List[str],
         :return: Dictionary of corrections grouped by ids of corresponding identifier \
                  in 'identifiers' and typoed tokens which have correction suggestions.
         """
-        df = pandas.DataFrame(columns=[self.config["index_column"], Columns.Split])
-        df[self.config["index_column"]] = range(len(identifiers))
-        df[Columns.Split] = [" ".join(self.parser.split(i)) for i in identifiers]
+        identifiers_positions = defaultdict(list)
+        for i, identifier in enumerate(identifiers):
+            identifiers_positions[identifier].append(i)
+        unique_identifiers = sorted(identifiers_positions.keys())
+        df = pandas.DataFrame(columns=[IDENTIFIER_ID_COLUMN, Columns.Split])
+        df[IDENTIFIER_ID_COLUMN] = range(len(unique_identifiers))
```

Comment:
well, you have `df.index` with the same values, I believe. 
Do you need this column or you can use `df.index` instead?

Reply:
No, I call `flatten` afterwards (to have one token per row instead of one identifier per row) and I need to store this `identifier id` separately to be able to combine suggestions for the tokens inside the same identifier.